common/src/main/java/com/linecorp/centraldogma/internal/Jackson.java

```diff
@@ -165,6 +167,15 @@ public static String writeValueAsPrettyString(Object value) throws JsonProcessin
         return compactMapper.convertValue(fromValue, toValueTypeRef);
     }
 
+    public static ObjectNode createObjectNode() {
+        compactMapper.createArrayNode();
+        return compactMapper.createObjectNode();
```

Comment:
You could just use `JsonNodeFactory.instance.*Node()`?

Reply:
Fixed. Thank you~!